sdks/java/core/pom.xml

```diff
@@ -430,6 +430,19 @@
 
     <!-- test dependencies -->
     <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-common-fn-api</artifactId>
+      <type>test-jar</type>
```

Comment:
Yes, I think this dependency is not appropriate for the SDK core because I think we should not be introducing any Fn-related deps here, even as an expedient measure since it could become a real dep accidentally.